### PR TITLE
CO-1643 : Add some tests for partner_compassion

### PR DESCRIPTION
**Pull request by samuel-fringeli**

### CODE REVIEW

File: partner_compassion/tests/test_partner_compassion.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            u'street': u'Impasse des Fr\xeanes 1',
+            u'zip': u'1669',
+            u'firstname': u'Samuel',
+            u'name': u'Fringeli Samuel',
+            u'city': u'Les Sciernes',
+            u'lastname': u'Fringeli',
+            u'ref': u'Fringeli',
+            u'church_id': self.church.id,
+            u'lang': u'en_US',
+            u'phone': u'+41 78 813 12 36'
```
Comment (ecino) [inline partner_compassion/tests/test_partner_compassion.py:36]: Do you really want to put your information on Github? I think random data should be used...

**Comment by ecino** [comment]
I guess that's a starting point... but coverage is not really changed with those small tests.

File: partner_compassion/tests/test_partner_compassion.py (hunk 1 above)
Comment (vwvw) [reply partner_compassion/tests/test_partner_compassion.py:36]: It happens even to the best ones ;)
https://github.com/CompassionCH/compassion-switzerland/blob/2845f2d19aa3be19680b9778dbca8efc07b372a1/wordpress_connector/models/contracts.py#L60-L71

Comment (ecino) [reply partner_compassion/tests/test_partner_compassion.py:36]: :laughing: It's my secret plan to become famous!
@samuel-fringeli Are you ok with this? If good, I will merge.

**Pull request merged**